bin/compile

```diff
@@ -372,6 +372,7 @@ for var in \`env | cut -f1 -d=\`; do
     echo "env[\$var] = \\$\${var}" >> /app/vendor/php/etc/php-fpm.conf
 done
 
+export APP_ENV="\${APP_ENV:-prod}"
```

Comment:
Can you add a comment stating that this variable is for Symfony apps?

Reply:
I can but the comment will be present in the file dropped in the user container.

Reply:
It's not a problem.

It's also used by laravel https://laravel.com/docs/5.7/configuration btw